automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/apps/AppLifecycleSpec.scala

```diff
@@ -62,23 +62,24 @@ class AppLifecycleSpec
     )
   )
 
-  // Test custom app first so that there will be a GKE cluster created already for the next two tests
```

Comment:
Hm, were we relying on a side effect of this test to create a GKE cluster before the subsequent (parallel) tests?

Reply:
Yeah, I was just thinking about that, I might make these two tests sequential instead of in parallel, but i was curious to see if this woks as is

Reply:
turns out it worked! Merging as is since I will have a follow up PR shortly reenabling the test with the new image anyways